quinn-proto/src/transport_parameters.rs

```diff
@@ -68,30 +68,30 @@ macro_rules! make_struct {
         /// Transport parameters used to negotiate connection-level preferences between peers
         #[derive(Debug, Copy, Clone, Eq, PartialEq)]
         pub struct TransportParameters {
-            $($(#[$doc])* pub(crate) $name : VarInt,)*
+            $($(#[$doc])* pub $name : VarInt,)*
 
             /// Does the endpoint support active connection migration
-            pub(crate) disable_active_migration: bool,
+            pub disable_active_migration: bool,
```

Comment:
I'm inclined to make methods for these so we can guarantee that these aren't modified from the outside, what do you think? @Ralith any opinions?

Reply:
@djc why not just rely on mutability of the struct?